specification/resourcehealth/resource-manager/readme.md

```diff
@@ -49,6 +49,15 @@ input-file:
 - Microsoft.ResourceHealth/stable/2015-01-01/resourcehealth.json
 ```
 
+### Tag: package-2018-08-preview
```

Comment:
should this tag be your default tag in line 29? or do you want to keep package-2017-07 as the default? The default tag is what's used to generate SDKs by default, unless you particularly specify and let us know something else.

Reply:
I am adding this new change to preview api. We will update api version to 2018-08-01 once it goes GA (basically not preview anymore). So will keep 2017-07 as default for now,.